### PR TITLE
P1 - List and Toast Components

### PR DESCRIPTION
**Pull request by bharati-21**
Styled and added a list component to the documentation. The following variations were added-
- spaced list
- stacked list

Styled and added a toast component to the documentation. The following variations were added-
- simple toast

### CODE REVIEW

File: Components/toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.snackbar {
+    background-color: var(--text-color);
+    color: #fff;
```
Comment (yashpatel1797) [inline Components/toast/toast.css:3]: used variable or hex only one.

**Review by yashpatel1797 — APPROVED** [review]
code looks good but in toast demo 2 things are getting overlapping sidebar, line and toast so you can check that.

**Review by AtulHande03a — APPROVED** [review]
Looks good and the class names are also well written!!

File: Components/toast/toast.css (hunk 1 above)
Comment (bharati-21) [reply Components/toast/toast.css:3]: Oh yes, thank you.

**Comment by bharati-21** [comment]
> code looks good but in toast demo 2 things are getting overlapping sidebar, line and toast so you can check that.

Yes, just noticed. Thank you for your review!

**Comment by bharati-21** [comment]
> Looks good and the class names are also well written!!

Thank you!

File: Components/toast/toast.css (hunk 1 above)
Comment (bharati-21) [reply Components/toast/toast.css:3]: Changed to variable.

**Pull request merged**